llmfoundry/models/layers/attention.py

```diff
@@ -5,13 +5,14 @@
 
 import math
 import warnings
-from typing import Any, List, Optional, Tuple
+from typing import Any, Dict, List, Optional, Tuple
```

Comment:
we can probably just use `dict` as the type instead of needing to import `Dict`
same thing can probably be said for `List` and `Tuple` (but if you want to say thats outside the scope of this PR thats fine)

Reply:
Fixed! (including the List and Tuple)